coreTable/OwidTable.ts

```diff
@@ -215,13 +215,11 @@ export class OwidTable extends CoreTable<OwidRow, OwidColumnDef> {
         return [min(mins), max(maxes)]
     }
 
-    filterByEntityName(name: EntityName) {
-        // todo; why not a filter by?
-        return new OwidTable(this.rowsByEntityName.get(name) || [], this.defs, {
-            parent: this,
-            tableDescription: `Filter out all entities except '${name}'`,
-            transformCategory: TransformType.FilterRows,
-        })
+    filterByEntityNames(names: EntityName[]) {
+        return this.filter(
+            (row) => names.includes(row[OwidTableSlugs.entityName]),
```

Comment:
Nice!

Do you know if the JS engine will optimize string[].includes ? If not may want to add a one liner above converting names into a set for perf.